js/ui/search.js

```diff
@@ -819,7 +819,6 @@ const ProviderIcon = new Lang.Class({
 
         this._content = new St.Widget({ layout_manager: new Clutter.BinLayout() });
         this._content.add_actor(icon);
-        this._content.add_actor(this.moreIcon);
```

Comment:
Please remove `moreIcon` altogether, instead of simply not adding it to the container.